docs/source/_static/js/custom.js

```diff
@@ -0,0 +1,4 @@
+// Open external links in a new tab
+$(document).ready(function () {
+    $('a[href^="http://"], a[href^="https://"]').not('a[class*=internal]').attr('target', '_blank');
```

Comment:
is this for autogenerated links? wondering why we cant add the target to any links we have in the docs.

Reply:
This is for any external links we use in RTD, for which we want to open the page in a new tab so that the user has RTD page in a tab to return to.